translations/OpenOrienteering_sv.ts

```diff
@@ -438,12 +438,12 @@
     <message>
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>CMYK</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
I need to test if empty entries result in empty or untranslated ouput. However, it is at least difficult to tell whether the missing entry is intentionally or by mistake.